napari_allencell_annotator/_tests/view/viewer_test.py

```diff
@@ -0,0 +1,84 @@
+import napari
+from napari.layers import Points
+
+from napari_allencell_annotator.view.viewer import Viewer, PointsLayerMode
+import numpy as np
+import pytest
+from napari.utils import Colormap
+
+
+@pytest.fixture
+def viewer(make_napari_viewer: napari.Viewer) -> Viewer:
+    return Viewer(make_napari_viewer())
+
+
+def test_get_all_points_layer(viewer: Viewer) -> None:
+    # ARRANGE
+    viewer.add_image(np.zeros(shape=(2, 2)))
+    test_points_layer1: Points = viewer.create_points_layer("test1", "blue", True)
+    test_points_layer2: Points = viewer.create_points_layer("test2", "blue", True)
+
+    # ACT
+    all_points_layer: list[Points] = viewer._get_all_points_layers()
+
+    # ASSERT
+    assert len(all_points_layer) == 2
+    assert test_points_layer1 in all_points_layer
+    assert test_points_layer2 in all_points_layer
+
+
+def test_add_image(viewer: Viewer) -> None:
+    # ACT
+    viewer.add_image(np.zeros(shape=(2, 2)))
+
+    # ASSERT
+    assert len(viewer.get_layers()) == 1
+    np.testing.assert_array_equal(viewer.get_layers()[0].data, np.zeros(shape=(2, 2)))
+
+
+def test_create_points_layer(viewer: Viewer) -> None:
+    # ACT
+    test_points_layer: Points = viewer.create_points_layer("test", "blue", True, np.zeros(shape=(1, 6)))
+
+    # ASSERT
+    assert test_points_layer in viewer._get_all_points_layers()
+    assert len(viewer._get_all_points_layers()) == 1
```

Comment:
public function

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/34cae500e0098750b6dec4da800ba0e7f3470a10